charts/ts-translation-service/values.preview.template.yaml

```diff
@@ -2,3 +2,8 @@ java:
   # Don't modify below here
   image: ${IMAGE_NAME}
   ingressHost: ${SERVICE_FQDN}
+  keyVaults:
+    ts:
+      secrets:
```

Comment:
Should there be a `resourceGroup` in between line 6 & 7? If so what would value be ?